src/Workspaces/Core/Portable/FindSymbols/SymbolFinder.cs

```diff
@@ -15,7 +15,11 @@ namespace Microsoft.CodeAnalysis.FindSymbols
     public static partial class SymbolFinder
     {
         /// <summary>
-        /// Finds the symbol that is associated with a position in the text of a document.
+        /// Finds the symbol that is associated with a position in the text of a document. If 
+        /// multiple symbols are associated with the token (e.g. implicitly named anonymous type
```

Comment:
@jasonmalinowski @CyrusNajmabadi Actually, regardless of whether we make a ```FindSymbolsAtPosition```, we'll still have this ```FindSymbolAtPosition``` to return _the_ symbol we think you probably want, which is what the method does now in this case. Let's discuss the potential ```FindSymbolsAtPosition``` API separately?